ansible-scylla-node/defaults/main.yml

```diff
@@ -260,6 +260,13 @@ skip_sysconfig: False
 skip_selinux: False
 skip_ntp: False
 
+# If defined, this enables the usage of a separate token_distritor instead of scylla's default random token distributor
```

Comment:
Is @tarzanek ok with such a way of handling dependencies? I am not proficient enough in ansible to suggest a better one, unfortunately

Reply:
so I don't think there are good ways to achieve this, let me try to find a way, but for now documenting is better than nothing

Reply:
typo in 
" token_distritor "

Reply:
Fixed